test/hotspot/jtreg/compiler/vectorization/runner/ArrayTypeConvertTest.java

```diff
@@ -161,10 +161,9 @@ public short[] narrowToSigned() {
     }
 
     @Test
-    @IR(failOn = {IRNode.STORE_VECTOR})
-    // Subword vector casts do not work currently, see JDK-8342095.
```

Comment:
There seem to be other cases in this file that mention `JDK-8342095`. We should probably file a new RFE for those, right?